05_wifi_virtualif.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/bash
+
+set -o nounset
+set -euo pipefail
+
+if [[ $EUID -ne 0 ]]; then
+	echo "This script must be run as root, use sudo $0 instead" 1>&2
+	exit 1
+fi
+
+## Install hostapd
+pacman -Syu --noconfirm hostapd
+
```

Comment:
NetworkManager randomises wifi interface MAC addresses by default. This messes up any attempts
to assign a fixed IP from the dhcp server based on the MAC address. Therefore, the mac randomisation should be disabled. Add the following snippet:
--8<--
## disable mac address randomisation
cat > /etc/NetworkManager/conf.d/30-mac-rnd.conf << EOF
# Disable wifi mac address randomisation
[device]
wifi.scan-rand-mac-address=no

EOF

systemctl restart NetworkManager.service
--8<--